src/config.rs

```diff
@@ -170,6 +170,27 @@ configuration_option_enum! { WriteMode:
     Checkstyle,
 }
 
+// How to reorder use statements.
+configuration_option_enum! { ImportReordering:
+    // Don't change `use` statements at all.
+    None,
+    // Just order `use` statements by their path.
+    Lines,
+    // Just order items within `use` statements by their name.
+    Items,
+    // Order `use` statements by their path and items within `use` statements by their name.
+    LinesAndItems
+}
```

Comment:
Absolutely - I had a look to see if there was a way that a list of enumerations could be supplied rather than just one (something like `reorder_imports = [ "Lines", "Imports" ]` or `reorder_imports = "Lines,Imports"`), but that was looking a bit involved, so I'll change to separate options.